content/cli/usages.md

```diff
@@ -89,6 +89,12 @@ $ nest g <schematic> <name> [options]
 
 Compiles an application or workspace into an output folder.
 
+Besides that, the `build` command is responsible for:
+
+- map paths properly for us (if using path aliases) via `tsconfig-paths`;
+- annotate DTOs with swagger decorators (if `@nestjs/swagger` is enable);
+- annotates DTOs with GraphQL decorators (if `@nestjs/graphql` is enable).
```

Comment:
```suggestion
- mapping paths properly for us (if using path aliases) via `tsconfig-paths`
- annotating DTOs with OpenAPI decorators (if `@nestjs/swagger` is enable)
- annotating DTOs with GraphQL decorators (if `@nestjs/graphql` is enable)
```

Following better grammar, "X is responsible for _doing_ Y"

Reply:
Also, `OpenAPI` vs Swagger for the "proper" name. Feel free to ignore that if we generally use swagger elsewhere in the docs